composio/cli/utils/helpfulcmd.py

```diff
@@ -33,7 +33,7 @@ def format_help_text(self, ctx: ClickContext, formatter: HelpFormatter) -> None:
     def format_options(self, ctx: ClickContext, formatter: HelpFormatter) -> None:
         """Writes all the options into the formatter if they exist."""
         opts = []
-        for param in self.get_params(ctx):
+        for param in self.get_params(ctx):  # type: ignore
```

Comment:
Hmmm seems a simple line, what's the issue.

Reply:
cc - @himanshu-dixit